splice_spark2/src/main/spark2.2/org/apache/spark/sql/execution/datasources/jdbc/SpliceRelation2.scala

```diff
@@ -0,0 +1,68 @@
+package org.apache.spark.sql.execution.datasources.jdbc
+
+import com.splicemachine.spark2.splicemachine.{SpliceJDBCOptions, SpliceJDBCUtil, SplicemachineContext}
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.jdbc.JdbcDialects
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.{DataFrame, Row, SQLContext, SparkSession}
+import org.apache.spark.sql.sources._
+
+/**
+  * Created by jleach on 4/7/17.
+  */
+
+case class SpliceRelation2(jdbcOptions: JDBCOptions)(@transient val sqlContext: SQLContext, @transient var userSchema: Option[StructType]) extends BaseRelation
```

Comment:
This has to be named as SpliceRelation2?

Reply:
Renamed it so it wouldn’t collide with SpliceRelation in the splice_spark folder.